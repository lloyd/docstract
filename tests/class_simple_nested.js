/** Simple test of nested class support */

/** @class Foo */
/** @constructor
 * foo's constructor! */
/** @class Bar */
/** @constructor
 * bar's constructor! */
/** @endclass */
/** @endclass */
