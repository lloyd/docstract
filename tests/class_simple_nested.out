{
  "classes": [
    {
      "classes": [
        {
          "constructors": [
            {
              "desc": "bar's constructor!", 
              "source_lines": [
                7, 
                8
              ]
            }
          ], 
          "name": "Bar", 
          "source_lines": [
            6, 
            6
          ]
        }
      ], 
      "constructors": [
        {
          "desc": "foo's constructor!", 
          "source_lines": [
            4, 
            5
          ]
        }
      ], 
      "name": "Foo", 
      "source_lines": [
        3, 
        3
      ]
    }
  ], 
  "desc": "Simple test of nested class support", 
  "filename": "tests/class_simple_nested.js", 
  "module": "class_simple_nested", 
  "source_lines": [
    1, 
    2
  ]
}
